src/components/NavBar/ChainSwitcher.tsx

```diff
@@ -72,24 +74,43 @@ export const ChainSwitcher = ({ isMobile }: ChainSwitcherProps) => {
     return null
   }
 
+  const isSupported = isChainAllowed(chainId)
+
   return (
     <Box position="relative" ref={ref}>
-      <Row as="button" gap="8" className={styles.ChainSwitcher} onClick={toggleOpen}>
-        <img src={info.logoUrl} alt={info.label} className={styles.Image} />
-        <Box as="span" className={subhead} style={{ lineHeight: '20px' }}>
-          {info.label}
-        </Box>
+      <Row
+        as="button"
+        gap="8"
+        className={styles.ChainSwitcher}
+        background={isOpen ? 'accentActiveSoft' : 'none'}
+        onClick={toggleOpen}
+      >
+        {!isSupported ? (
+          <>
+            <TokenWarningRedIcon fill={themeVars.colors.darkGray} width={24} height={24} />
+            <Box as="span" className={subhead} style={{ lineHeight: '20px' }}>
+              {info?.label ?? 'Unsupported'}
```

Comment:
if the chain is unsupported it won't be in info, so can skip that. also should add a translate wrapper on this

Reply:
Localization for the Navbar will be handled in a subsequent PR

Reply:
i think this part does need to change though, since the info label case will never be hit